File: 07. Perulangan Lanjut/Break Continue Exit/BreakContinueExit.java
import java.util.Scanner;

public class BreakContinueExit
{
    public static void main (String [] args)
    {
        Scanner scanner = new Scanner (System.in);

        short bilangan = scanner.nextShort ();

        scanner.close ();

        for (short i = 1; i <= bilangan; i++)
        {
            if (i % 10 == 0)
            {
                continue;
            }

            if (i == 42)
            {
                System.out.print("ERROR");
                break;
            }

            System.out.println (i);
        }
    }
}

File: 07. Perulangan Lanjut/Break Continue Exit/Break-Continue-Exit.cpp
#include <iostream>

int main ()
{
    unsigned short bilangan = 0;

    std::cin >> bilangan;

    for (unsigned short i = 1; i <= bilangan; i++)
    {
        if (i % 10 == 0)
        {
            continue;
        }
        
        if (i == 42)
        {
            std::cout << "ERROR";
            break;
        }

        std::cout << i << "\n";
    }

    return 0;
}